code/Controllers/CMSPagesController.php

```diff
@@ -31,35 +31,4 @@ public function isCurrentPage(DataObject $record)
     {
         return false;
     }
-
-    public function Breadcrumbs($unlinked = false)
-    {
-        $this->beforeExtending('updateBreadcrumbs', function (ArrayList $items) {
-            //special case for building the breadcrumbs when calling the listchildren Pages ListView action
-            if ($parentID = $this->getRequest()->getVar('ParentID')) {
-                $page = SiteTree::get()->byID($parentID);
-
-                //build a reversed list of the parent tree
-                $pages = [];
-                while ($page) {
-                    array_unshift($pages, $page); //add to start of array so that array is in reverse order
-                    $page = $page->Parent;
-                }
-
-                //turns the title and link of the breadcrumbs into template-friendly variables
-                $params = array_filter([
-                    'view' => $this->getRequest()->getVar('view'),
-                    'q' => $this->getRequest()->getVar('q')
```

Comment:
These parameters are useless ... I couldn't find anyway to get the CMS to output them for me. Even if I manually put them in, nothing happens with them. My guess is that they are a left over from a time someone was hoping to allow the search to work in conjunction with a tree view or something.

I just took them out from my final implementation.

Reply:
Please reinstate them - those aren't related to the problem you're fixing and since we don't really know what they're doing, removing them now could introduce unexpected regressions. We can remove them in a future refactor after looking into it further.

Reply:
Fair enough. I'll reinstate that bit of logic.